src/main.js

```diff
@@ -41,9 +46,26 @@ function saveTask(title) {
   localStorage.setItem("tasks", tasks);
 }
 
+function pendingTasks(){
+  const storedTasks = localStorage.getItem("tasks");
+  let storedTasksArray = [];
+  console.log(storedTasks);

```

Comment:
Remove `console.log` from the code. It should only be used for debugging

Reply:
done